playbooks/vars/defaults.yaml

```diff
@@ -145,6 +145,16 @@ kernel_services:
 #sriov_nic_numvfs:
 #sriov_nova_pci_passthrough:
 
+
+# For pci passthrough provide the relevant fields. for example:
+#pci_passthrough_devices:
```

Comment:
this is not enough, when configuring NovaPCIPassthrough you need to add a filter to Nova, which is done via /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovn-sriov.yaml (automatically loaded when setting `sriov_interface`).

So maybe we could add a conditional in `Add sriov to enabled services` in `playbooks/install_stack.yaml` to automatically load that file, so we get the right Filters configured in Nova. However, it'll also configure Neutron ML2 mechanism driver to ['sriovnicswitch','ovn'], I don't know if this is a problem if we don't configure SR-IOV interface.

Reply:
right this is not enough, thus why the WIP in the title.

Reply:
This no longer applies